### PR TITLE
Use megabytes for upload file size limit

### PR DESCRIPTION
**Pull request by koladeadewuyi**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -180,7 +180,7 @@ features {
 
 upscan {
   callbackUrl = "http://localhost:12600/upscan-callback"
-  maxFileSize = "5 MiB"
+  maxFileSize = "5 MB"
   maxFiles    = 5
 }
```
Comment (amaalali) [inline conf/application.conf:185]: Is there a reason that we have what seems to be the same value in two places?

**Review by amaalali — CHANGES_REQUESTED** [review]
I have a quick question about a potential duplication in config values

**Review by C-Thomp — APPROVED** [review]
👍 working as expected

File: conf/application.conf (hunk 1 above)
Comment (C-Thomp) [reply conf/application.conf:185]: ones for upscan and the other is for validation - though i think i see what you mean

Comment (koladeadewuyi) [reply conf/application.conf:185]: Not sure I understand what you mean @amaalali. The values are different.

Comment (amaalali) [reply conf/application.conf:185]: Totally misread this, I thought that they were the value and human readable version

**Review by amaalali — APPROVED** [review]

**Pull request merged**